projects/js-packages/publicize-components/src/social-store/selectors/shares-data.ts

```diff
@@ -85,17 +85,10 @@ export function getTotalSharesCount(
 ) {
 	let count = getSharesUsedCount( state ) + getScheduledSharesCount( state );
 
-	if ( options.excludeInitialConnections ) {
-		count -= getInitialEnabledConnectionsCount( state );
-	}
-
-	if ( ! options.includeEnabledConnections ) {
-		return count;
-	}
-
-	const enabledConnections = getEnabledConnections( state );
+	count -= options.initialEnabledConnectionsCount ?? 0;
```

Comment:
Again, I'm trying move this logic and calculations to the hooks. I think we might not need `getTotalSharesCount` at all now